### ISSUE TITLE
Update the value of access flag ACC_VALUE

### ISSUE THREAD

**Issue by hangshao0** [eclipse-openj9/openj9#15081]
There is an update to the value object [JEP](https://openjdk.java.net/jeps/8277163) yesterday that class access flag `ACC_VALUE` is changed from `0x0100` to `0x0040`. 

Once the corresponding code change is made by OpenJDK and merged into https://github.com/ibmruntimes/openj9-openjdk-jdk.valuetypes, OpenJ9 needs to change 
https://github.com/eclipse-openj9/openj9/blob/e9e305a509717a53025048f4f87b73c12f2a4142/runtime/oti/cfr.h#L861 to `0x0040`.

There are more change from http://cr.openjdk.java.net/~dlsmith/jep8277163/jep8277163-20220519/specs/value-objects-jvms.html. 

In summary, the changes are:

- [x] update the value of `ACC_VALUE` (#15130)
- [x] `ACC_PERMITS_VALUE` is removed #15130)
- [x] `ACC_IDENTITY (0x0020)` is added, it reuses the value of `ACC_SUPER` (#15130)
- [x] `java/lang/IdentityObject` interface is removed (#15130)
- [x] The unnamed factory method `<new>` is renamed to `<vnew>` (OpenJDK has not switched to `<vnew>` yet, still using `<init>`) (issue: https://github.com/eclipse-openj9/openj9/issues/14959)

**Comment by hangshao0** [comment]
A bit more updates I noticed:

`ACC_PERMITS_VALUE` is removed and `ACC_IDENTITY (0x0020)` is added.
The `java/lang/IdentityObject` interface is removed. 
The unnamed factory method `<new>` is renamed to `<vnew>`

**Comment by hangshao0** [comment]
> ACC_IDENTITY (0x0020) is added, it reuses the value of ACC_SUPER

OpenJDK has added `ACC_IDENTITY`, but its asm code is not updated yet. Currently it does not allow `ACC_IDENTITY` to be set, which is causing some of our tests to fail:

```
java.lang.IllegalArgumentException: Invalid access flags: 58
	at java.base/jdk.internal.org.objectweb.asm.util.CheckClassAdapter.checkAccess(CheckClassAdapter.java:573)
	at java.base/jdk.internal.org.objectweb.asm.util.CheckClassAdapter.visitInnerClass(CheckClassAdapter.java:395)
	at java.base/jdk.internal.org.objectweb.asm.ClassReader.accept(ClassReader.java:749)
	at java.base/jdk.internal.org.objectweb.asm.ClassReader.accept(ClassReader.java:456)
	at org.openj9.test.javaagenttest.util.Transformer.transform(Transformer.java:69)
	at org.openj9.test.javaagenttest.RefreshGCCache_ExtendedHCR_Test.test_EXTENDED_HCR(RefreshGCCache_ExtendedHCR_Test.java:49)
```

We need to exclude these tests in vt standard build until OpenJDK update asm to include `ACC_IDENTITY`.

**Comment by hangshao0** [comment]
This issue can be closed.

**Comment by pshipton** [comment]
@hangshao0 can the `test excluded` label be removed from this issues since we have https://github.com/eclipse-openj9/openj9/issues/12878 for tracking? We shouldn't have any closed issues with `test excluded`.

**Comment by hangshao0** [comment]
> can the test excluded label be removed from this issues

Yes. I will remove it.
